packages/dashboard-frontend/stories/task-list/task-progressbar/tasks-progressbar.stories.js

```diff
@@ -0,0 +1,38 @@
+import { TasksProgressBar } from "../../../src/task-list/components/tasks-progressbar";
+import documentation from "./documentation.md";
```

Comment:
`packages/dashboard-frontend/stories/task-list/task-progressbar/documentation.md` is empty.